pkg/lang/javascript/expose.go

```diff
@@ -110,7 +109,7 @@ func handleGatewayRoutes(info *execUnitExposeInfo, constructGraph *graph.Directe
 				// TODO when there are ways to combine units, we'll need a more sophisticated way to see which unit the target maps to.
 				depKey.ID = info.Unit.ID
 			}
-			constructGraph.AddEdge(gw.Provenance().ToString(), depKey.ToString())
+			constructGraph.AddEdge(gw.Id(), depKey.ToString())
```

Comment:
Ah, now I see why `ToString` was added. Usually I'd associate the `ToString` with textual representation meant for humans (it's the `Stringer` interfaced and used in `fmt.Print` functions). I have a slight preference to name this more like `ToId`.

Reply:
Thats fair, i think we can rename that

Reply:
renamed in the most recent commint